tests/unit-tests/python/foglamp_test/common/test_service_deregister_on_fail.py

```diff
@@ -19,12 +21,12 @@
 
 name = "Foo"
 core_host = "localhost"
-core_port = "37061"
+core_port = "36793"
 
 
-@pytest.allure.feature("common")
-@pytest.allure.story("process")
-class TestMicroservice:
+@pytest.allure.feature("core")
+@pytest.allure.story("monitor")
+class TestMonitoring:
```

Comment:
quick fix; FOGL-922 will handle it better; also need to see why ping does not work for `foo.py`